MyFirstSlideshow/app/build.gradle

```diff
@@ -20,4 +20,5 @@ android {
 
 dependencies {
     compile 'com.android.support:appcompat-v7:25.3.1'
+    compile 'com.karumi:dexter:4.1.0'
```

Comment:
Do not use "compile" keyword. Use "implementation" keyword instead. It works same but "compile" is deprecated for some time.